### PR TITLE
Add an HTML and its corresponding CSS file

### PR DESCRIPTION
**Pull request by bizip**
- Add project startup files

- Add HTML and CSS files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <div id="header">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+        </div>
+
+
+        <div id="img-div">
+            <img id="image"
+                src="https://images.unsplash.com/photo-1613837433169-d74597534a69?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTR8fGRyLiUyMG5vcm1hbiUyMGJvcmxhdWd8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60"
+                alt="image" />
```
Comment (bizip) [inline index.html:23]: Kindly, try to write something inside atl attribute in the image tag that is not the image keyword, in order to allow screen readers to clarify what to say.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <div id="main">
+        <div id="header">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+        </div>
+
+
+        <div id="img-div">
+            <img id="image"
+                src="https://images.unsplash.com/photo-1613837433169-d74597534a69?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTR8fGRyLiUyMG5vcm1hbiUyMGJvcmxhdWd8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60"
```
Comment (bizip) [inline index.html:22]: Please, try to break Code with long lines into short lines of code to enhance the readability.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+#main {
+  background-color: silver;
+}
+
+#main h2 {
```
Comment (bizip) [inline main.css:10]: Kindly, use  another alternative as a CSS selector than ID eg: classes and HTML tags, etc

**Review by bizip — COMMENTED** [review]
# Status: Required Changes ♻️ 

Hi @bizip, 

Alt Text

Good job so far! 👏🏾 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ All the tests are passing
✔️Code is organized and readable

## Required Changes ♻️
- [ ] Kindly write some text inside alt attribute that is not a keyword
- [ ] Please use alternative CSS selector than ID


## Optional suggestions
To enhance readability kindly use a short line of code

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@bizip` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
Hi @bizip, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**